docs-kits/kits/Circularity_KIT/page-adoption-view.md

```diff
@@ -445,10 +476,77 @@ The core process means in a chronological order: The basic reuse-potential, mate
+
+-	As a (Core) Purchaser, I want to search and filter for market supply on OE number level to match the market offers towards my demand.
+-	As a (Core) Purchaser, I want to reach as many possible sellers as possible to purchase parts that I cannot procure today. This includes brokers I don't buy from today.
+-	As a (Core) Seller, I want to place offers on the marketplace with standard upload tools and manually.
+-	As a (Core) Seller, I want to reach as many possible customers as possible to maximize my turnover. This includes customers I don't sell to today.
+
+The key user journey relevant in Catena-X circular economy area relates to the need of a buyer to have full reliable information on a product that he is intending to purchase. This is especially important given that many buyers are looking for parts for re-manufacturing or re-use, where product quality plays a vital role. Furthermore, in case of recycling, recyclers are interested to know material composition and additional information that would influence their purchasing decision. 
+
+Some information could be provided by a variety of data models, but the highest value lies within the generic digital product passport (and by extension, battery passport, sealant passport, transmission passport). When a seller is publishing a listing with a product that he is willing to sell, there is only a limited amount of information he can provide with it, mostly the one available from the visual inspection or on-board data (such as mileage, OE-number, brand) as well as logistical/ sales information (such as location or price). When a potential buyer views the listing, he should have a possibility to request additional details from a data provider of the generic digital product passport (if available) for that product (see diagram below). More information on the digital product passport can be found in the [Ecopass KIT](https://eclipse-tractusx.github.io/docs-kits/kits/Eco_Pass_KIT/page-adoption-view/).
+
+**Figure 13**
+
+![User Journey - Secondary Marketplace](resources/adoption-view/user_journey_secondary_marketplace.svg)
+
+This process is reflected in the architecture for communicating with the Catena-X network and data providers of the battery passport data model, as presented below.
+
+**Figure 14**
+
+![Communication Architecture](resources/adoption-view/communication_architecture.svg)
+
+When it comes to the data exchange schema and other details, see the [Triangle for Secondary Marketplace](https://catena-x.net/de/standard-library) document for more information on how such a request could be realized within Catena-X environment and according to applicable guidelines (e.g. on interoperability and data sovereignty). In a decentral environment, such attributes as manufacturer part ID and OE number play a vital role in order to get access to the right component and the right data model at data provider’s side.
+
+### Semantic Models
```

Comment:
We would suggest to create a separate Standard Section for all your standards.
In general, please add the valid standard version to all of your mentioned standards.